README.md

```diff
@@ -16,6 +16,8 @@ aiken,spend,pass,15,517656,2001
 eopsin,spend,pass,160,2045491,7702
 hebi,spend,pass,28,713100,3200
 helios,spend,pass,8,230100,1100
+helios,spend,pass,8,230100,1100
+plu-ts,spend,pass,8,230100,1100
```

Comment:
I would prefer if you add a simple always-succeed example contract and run it before listing it the README.

You should be able to basically just copy&paste the contract that you just added